common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -336,4 +345,26 @@ public static JsonObject getJsonObjectFromString(String jsonString) {
         return JsonParser.parseString(jsonString).getAsJsonObject();
     }
 
+    public static void validateSchema(String schemaString, String instanceString) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
This would create a new instance of ObjectMapper every time when this function is called, which is not efficient. I suggest we become cautious about "new Object()" statement in utility method. So it's better to use a shared, thread-safe instance like this in the class.

private static final ObjectMapper MAPPER = new ObjectMapper();